berth_reservations/settings.py

```diff
@@ -342,3 +344,6 @@
     exec(code, globals(), locals())
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+# Monkey-patching
```

Comment:
Could be highlighted even more! And should be mentioned clearly in the change logs.